### PR TITLE
Completed html-skeleton-excercise.

### PR DESCRIPTION
**Pull request by phong4**
Screen Shot 2021-11-29 at 11 54 57 AM

### CODE REVIEW

File: html-skeleton/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML>
+<html>
+
+<head></head>
+<title>HTML Skeletons</title>
```
Comment (codingcodymiller) [inline html-skeleton/index.html]: The `title` element needs to be a child of the `head` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-skeleton/index.html (hunk 1 above)
Comment (phong4) [reply html-skeleton/index.html]: Hi @codingcodymiller I have made the change. Thanks!

File: html-skeleton/index.html [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
 <html>
-
-<head></head>
-<title>HTML Skeletons</title>
-
-<body>Hello, HTML!</body>
-
+  <head>
+    <title>HTML Skeletons</title>
+  <body>Hello, HTML!</body>
```
Comment (codingcodymiller) [inline html-skeleton/index.html]: The `body` element needs to be a sibling to `head`, not a child of it

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-skeleton/index.html (hunk 2 above)
Comment (phong4) [reply html-skeleton/index.html]: I have made the changes @codingcodymiller

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**